internal/stack/_static/docker-compose-stack.yml.tmpl

```diff
@@ -112,6 +112,10 @@ services:
       - "../certs/fleet-server:/etc/ssl/elastic-agent"
     ports:
       - "127.0.0.1:8220:8220"
+      {{ $apm_server_enabled := fact "apm_server_enabled" }}
+      {{ if eq $apm_server_enabled "true" }}
+      - "127.0.0.1:8200:8200"
+      {{ end }}
```

Comment:
Good idea to add this to the Fleet Server agent :+1: maybe at some point we can keep it always enabled and/or reuse it for actual performance monitoring of the testing framework.

Another option would be to create another agent, but this will probably bring unnecessary complexity.